docs/docs/experiments.mdx

```diff
@@ -47,14 +47,14 @@ handle all of this for you, and you can also serve client-side A/B test via our
 Client-side A/B tests are great for testing visual changes to your application, but they do have some drawbacks. One of
 the most common issues is caused by the delay in loading the specific variation to a user, which may cause a flash or
 flickering as the experiment loads. This can be reduced by using inline-experiments, or by moving the GrowthBook SDK
-code higher in the code so it loads at the same time or before the page. Sometimes the hashing attribute may not be
+code higher in the code so it loads at time or before the page. Sometimes the hashing attribute may not be
```

Comment:
you introduced another typo?

Reply:
i'm baffled and am open to having made this mistake but really don't think i changed anything here

Reply:
someone with better git-fu please fix and commit, i have limited permissions